openapi/components/schemas/GatewayAccountConfig/Payretailers.yaml

```
@@ -0,0 +1,20 @@
+description: Payretailers gateway configuration.
```

suggestion: I think their branding capitalizes the 'R' in Retailers
```suggestion
description: PayRetailers gateway configuration.
```